### PR TITLE
Feature/serialize json out

### PR DESCRIPTION
**Pull request by CloudNiner**

### CODE REVIEW

File: app/api/serializer_fields.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    Do no transformations, as we always want python dicts
+    Validate schema when converted to internal representation
+
+    """
+    type_name = 'JsonBField'
+
+    def to_representation(self, value):
+        return value
+
+    def to_internal_value(self, value):
```
Comment (ddohler) [inline app/api/serializer_fields.py:20]: I don't think this is appropriate on the field; `check_schema()` will validate that the JSON is a valid JSON-Schema definition, which not all JsonBFields will be.

Comment (ddohler) [inline app/api/serializer_fields.py:20]: We could make a `JsonSchemaField` that inherits from this though and this would work for that.

Comment (CloudNiner) [inline app/api/serializer_fields.py:20]: Good point. And yup, that's exactly what I'll do.

Comment (CloudNiner) [inline app/api/serializer_fields.py:20]: Hmm, I'll probably also require that `value` here be an instance of list or dict.

Comment (ddohler) [inline app/api/serializer_fields.py:20]: Yeah that makes sense. Then RecordSerializer can have a JsonBField, and RecordSchemaSerializer can have a JsonSchemaField.

**Comment by ddohler** [comment]
+1

**Comment by CloudNiner** [comment]
Updated. I'll add some tests tomorrow am.

File: app/api/serializer_fields.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+            raise ValidationError('Array or object required')
+
+
+class JsonSchemaField(JsonBField):
+    """ Extend JsonBField to validate on conversion """
+
+    type_name = 'JsonSchemaField'
+
+    def to_internal_value(self, value):
+        try:
```
Comment (ddohler) [inline app/api/serializer_fields.py:34]: Call `__super__` here--shouldn't actually change anything but might result in a nicer error message. No idea what happens if you pass non-dicts to the jsonschema functions.

**Comment by CloudNiner** [comment]
One more time, updated w/ `super()` call and tests

**Comment by ddohler** [comment]
Tests look good, +1

**Pull request merged**